nullaway/src/main/java/com/uber/nullaway/handlers/GuavaAssertionsHandler.java

```diff
@@ -49,6 +65,11 @@ public MethodInvocationNode onCFGBuildPhase1AfterVisitMethodInvocation(
       } else if (callee.name.equals(checkStateMethod)) {
         phase.insertThrowOnFalse(originalNode.getArgument(0), preconditionCheckStateErrorType);
       }
+    } else if (callee.enclClass().getQualifiedName().equals(verifyClass)
+        && !callee.getParameters().isEmpty()) {
+      if (callee.name.equals(verifyMethod)) {
```

Comment:
Are we separating this conditional in two for the sake of future `Verify.*` methods we might support?

Reply:
Ah, you're right.  Will combine these and then land.